demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -259,6 +269,34 @@ def update_tests(self) -> None:
                 else:
                     logger.debug(f'Not formatting {self.source_file} with "No tests"')
 
+    def remove_from_conf_json(self, file_type, content_item_id) -> None:
```

Comment:
Let's add test to the format command as well. 
when using the -d flag, that it does what it suppose to do.